public/js/repeater.js

```diff
@@ -0,0 +1,185 @@
+window.Repeater = (function() {
```

Comment:
Also cool - since we didn't have webpack in the passport app, I was using old-style modularization, where we expose a single global variable per module. But you're right, now that we have webpack, that's the right way to do it :]